src/Microsoft.IdentityModel.JsonWebTokens/JwtRegisteredClaimNames.cs

```diff
@@ -135,7 +133,6 @@ public struct JwtRegisteredClaimNames
         public const string Nbf = "nbf";
 
         /// <summary>
-        /// http://tools.ietf.org/html/rfc7519#section-4
         /// </summary>
         public const string Prn = "prn";
```

Comment:
Could you please include this as the summary comment for the claims that aren't registered?

"Common claim type that's not currently registered."

Reply:
Yeah. No problem. Will include those

Reply:
@duki994 what type of guitar do you play? Do you have a favorite?